protobuf/proto/health.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";

```

Comment:
We want to create it independently and not add to the existing services?

Reply:
The standard says that the health check service should be in the 'grpc.health.v1' package, so I think this is the only way I can do that, no?